pkg/sync/v5/data/ConfigMap/openshift-azure-logging/fluentd-config.yaml

```diff
@@ -9,19 +9,58 @@ data:
       </storage>
       tag logs
     </source>
-    <match logs>
+     # drop unused columns
+    <filter logs>
+      @type record_transformer
+      remove_keys _BOOT_ID,_CAP_EFFECTIVE,_MACHINE_ID,_SOURCE_REALTIME_TIMESTAMP,_STREAM_ID,_TRANSPORT,_SOURCE_MONOTONIC_TIMESTAMP
+    </filter>
+    # split logs to generic, audit and kubernetes runtime logs
+    <match logs >
       @type rewrite_tag_filter
       <rule>
         key MESSAGE
         pattern audit\.k8s\.io
         tag audit
       </rule>
+      <rule>
+        key CONTAINER_NAME
+        pattern ^k8s_
+        tag kubernetes
+      </rule>
       <rule>
         key MESSAGE
         pattern .+
         tag journald
       </rule>
     </match>
+    # filter kubernetes runtime logs for transformation
+    <filter kubernetes >
+      @type kubernetes_metadata
```

Comment:
I was hoping we could use regexps: no dependency on kubernetes actually being up, allows us to remove the reader roles

Reply:
This is already part of td-agent image and it makes code simpler. Considering its ruby my vote is for "easier to read", as debugging issues with fluentd and ruby is pain :/ 

But if you want we can do it but it will not make it less complex.

Reply:
I want to use regexps so that there is no dependency on apiservers and etcd to be up for our logs to work.

Reply:
this filter should be removed now, right?

Reply:
will remove it. Need to sort out regexps.